R/source.R

```diff
@@ -15,10 +15,18 @@
 #' @export
 source_python <- function(file, envir = parent.frame(), convert = TRUE) {
   
+  # Copy lines from URL to a local tempory file
+  if (!file.exists(file) && isTRUE(grepl("http", file))) {
+    lines <- readLines(file, warn = FALSE)
```

Comment:
Why not use `download.file` (http://stat.ethz.ch/R-manual/R-devel/library/utils/html/download.file.html) here to just directly download the full file contents (as opposed to the lines treatment) e.g.

```r
tmpfile <- tempfile()
download.file('https://www.google.com', tmpfile)
```

Reply:
Thanks! Just modified the code to use this instead!